cluster/common.sh

```diff
@@ -603,6 +603,7 @@ KUBELET_TOKEN: $(yaml-quote ${KUBELET_TOKEN:-})
 KUBE_PROXY_TOKEN: $(yaml-quote ${KUBE_PROXY_TOKEN:-})
 ADMISSION_CONTROL: $(yaml-quote ${ADMISSION_CONTROL:-})
 MASTER_IP_RANGE: $(yaml-quote ${MASTER_IP_RANGE})
+MASTER_RESERVED_IP: $(yaml-quote ${MASTER_RESERVED_IP})
```

Comment:
The question is what will be the cost (=loss) on progression from (1) to (2). We estimate it using this [trix](https://docs.google.com/spreadsheets/d/1svlSnQehu0Vae0j9vwWtvLHm1-74-Da0a-lND2U_b3w/edit#gid=0&vpid=A1). According to it, in 3 nodes cluster, communication through the external IP will increase total cluster cost by **0.03%**.

In my opinion the loss is acceptable, @roberthbailey WDYT?